### ISSUE TITLE
Various Items Throw Without a DisplayData Attribute

### ISSUE THREAD

**Issue by StrangeWill** [lcourson/Hangfire.Dashboard.Management.v2#2]
Various lines such as this:
https://github.com/lcourson/Hangfire.Dashboard.Management.v2/blob/master/Pages/ManagementBasePage.cs#L416

`inputs += InputTextbox(myId, displayInfo.CssClasses, labelText, placeholderText, displayInfo?.Description, displayInfo.DefaultValue, displayInfo.IsDisabled);`

Require `displayInfo.DefaultValue`, so "simple jobs" without the attribute just throw a null ref. I'd make a PR but I'm in the middle of putting out a fire related to getting this working -- may submit one later today.

**Comment by lcourson** [comment]
@StrangeWill 
I am not sure I understand what you are seeing.  I have updated the master branch to include some examples of using this code.  Could you please pull the latest and find out if you are still having issues?  If so, please provide me a repo where it is reproduced and I will take a look.

**Comment by StrangeWill** [comment]
It may be a bad on my part assuming that `DisplayData` should be optional (since a lot of things use `displayInfo?` and some previous checks assume it can be null), but if I do this:


```csharp
namespace ASP.Net_Web_Application.HangfireManagement
{
	[ManagementPage(MenuName = "Expedited Jobs", Title = nameof(Expedited))]
	public class Expedited : IJob
	{
		[DisplayName("Job Number 1")]
		[Description("This is the description for Job Number 1")]
		[Queue("expedited")]
		public void Job1(PerformContext context, IJobCancellationToken token, string strInput1)
		{
			//Do awesome things here
		}
	}
}
```

I'll get a null ref because `DisplayData` attributes are _required_ on strings (and is missing from `strInput1`).

Either I'd expect a better checked/thrown exception that `DisplayData` is required or this:

```csharp
inputs += InputTextbox(myId, displayInfo.CssClasses, labelText, placeholderText, displayInfo?.Description, displayInfo.DefaultValue, displayInfo.IsDisabled);
```

Should be:
```csharp
inputs += InputTextbox(myId, displayInfo?.CssClasses, labelText, placeholderText, displayInfo?.Description, displayInfo?.DefaultValue, displayInfo?.IsDisabled ?? false);
```


---

I know it's kind of silly, but I've just thrown this on an already existing project to access some calls, while others are left fairly unattributed.

**Comment by massada** [comment]
I've seen this issue as well. This throws a NullReferenceException. I've open a PR.

**Comment by lcourson** [comment]
PR has been merged and NuGet package has been updated to 2.0.4 to incorporate the changes from @massada.  Thanks!
